build/main.rs

```diff
@@ -53,9 +52,10 @@ fn main() {
         println!("Failed to run rustfmt, will skip formatting generated files.")
     }
 
-println!("cargo:rerun-if-changed=cbindgen.toml");
-let manifest_dir = std::env::var("CARGO_MANIFEST_DIR").unwrap();
-cbindgen::generate(&manifest_dir).unwrap().write_to_file("path/to/file.h")
+    println!("cargo:rerun-if-changed=.cbindgen.toml");
+    let manifest_dir = std::env::var("CARGO_MANIFEST_DIR").unwrap();
+    cbindgen::generate(&manifest_dir).unwrap().write_to_file("docs/shared_library/libscryer_prolog.h");
```

Comment:
You probably want the file path passed to `write_to_file` be relative to the path in 
 `manifest_dir`